### DISCUSSION TITLE
Track slider response time

### DISCUSSION THREAD

**Discussion by Vasmares** [BimmerGestalt/AAIdrive#671] [Q&A]
Hi,

last year I metioned that on iOS one had the ability to move the slider of the current track.
It was implemented on the spot. 
Thanks again.

Now I was wondering.

I seem to remember it being more responsive. 
Now you can "tap" on a certain spot on the slider and the track will jump to that timestamp.

Is there a way to make it "draggeble" ?

I tried doing so, but it seems to have an severe "input lag".
When I dragged over the slider, the track jumped to a few time stamps, but only ever second as it seems.
Is there some kind of refresh rate for the UI ?

![grafik](https://user-images.githubusercontent.com/10863304/218263118-6296ef2c-a6a8-4409-9bb0-9729826593c7.png)

**Comment by hufman** [comment]
In the original implementation, AAIdrive had some blocking elements in the redraw loop that caused some input lag, and so the progress bar touch handler only [allowed input every 0.5 seconds](https://github.com/BimmerGestalt/AAIdrive/blob/59eff043ea01227a7553e28578526ca5ea56cb47/app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/PlaybackView.kt#L218). This could probably be tightened now that the redraw loop has pushed most things to async updates, let me try that out!

**Reply by hufman** [reply]
Turns out my quick tweak to change the debounce threshold didn't fix it, so drag support will take longer than expected.

**Reply by Vasmares** [reply]
Would you mind looking further ?
Would be a nice addon.
